Northstar.Client/mod/scripts/vscripts/ui/menu_private_match.nut

```diff
@@ -575,17 +565,12 @@ function UpdatePrivateMatchButtons()
 		Hud_SetLocked( file.selectMapButton, true )
 		Hud_SetLocked( file.selectModeButton, true )
 		Hud_SetLocked( file.matchSettingsButton, true )
-		
-		if ( !IsNorthstarServer() )
-			Hud_SetLocked( file.inviteFriendsButton, true )
 	}
 	else
 	{
 		RHud_SetText( file.startMatchButton, "#START_MATCH" )
 		Hud_SetLocked( file.selectMapButton, false )
 		Hud_SetLocked( file.selectModeButton, false )
-		if ( !IsNorthstarServer() )
-			Hud_SetLocked( file.inviteFriendsButton, false )
```

Comment:
Need to double check in testing but wouldn't that mean that the button is unlocked but then inviting someone actual will just have no effect?

Reply:
In testing the button just doesn't exist, same as before.
In the lines of code above this, the combo button for inviting friends is added, under the same condition (`!IsNorthstarServer()`) which has also been removed in this PR